library/src/main/java/io/github/boguszpawlowski/composecalendar/month/MonthContent.kt

```diff
@@ -66,25 +68,36 @@ internal fun <T : SelectionState> MonthPager(
     )
   }
 
-  LazyRow(
-    modifier = modifier.testTag("MonthPager"),
-    state = listState,
-    flingBehavior = flingBehavior,
-    verticalAlignment = Alignment.Top,
-  ) {
-    items(PagerItemCount) { index ->
-      MonthContent(
-        modifier = Modifier.fillParentMaxWidth(),
-        showAdjacentMonths = showAdjacentMonths,
-        selectionState = selectionState,
-        currentMonth = monthListState.getMonthForPage(index),
-        today = today,
-        daysOfWeek = daysOfWeek,
-        dayContent = dayContent,
-        weekHeader = weekHeader,
-        monthContainer = monthContainer
+  Column(modifier = Modifier.fillMaxWidth()) {
+    if (!scrollWeekDays) {
```

Comment:
I would flip the condition to not have negation in there, as it is harded to read. 
```suggestion
    if (scrollWeekDays) {
```

Reply:
There's no else block here - do you want me to switch it with .not() call so it becomes easier to read?